hardware_integration/src/innomaker_usb2can_windows_plugin.cpp

```diff
@@ -238,7 +238,7 @@ namespace isobus
 
 				default:
 				{
-					isobus::CANStackLogger::error("[InnoMaker-Windows] Unsupported baudrate with index " + isobus::to_string(baudrate) + " in InnoMakerUSB2CANWindowsPlugin::Baudrate enum.");
+					LOG_IF_ENABLED(isobus::CANStackLogger::error("[InnoMaker-Windows] Unsupported baudrate with index " + isobus::to_string(baudrate) + " in InnoMakerUSB2CANWindowsPlugin::Baudrate enum."));
```

Comment:
```suggestion
					LOG_ERROR("[InnoMaker-Windows] Unsupported baudrate with index " + isobus::to_string(baudrate) + " in InnoMakerUSB2CANWindowsPlugin::Baudrate enum."));
```
thought: could we achieve something like this? And then for each level a different define

Reply:
Probably, that would be much shorter which is nice. I can work on that, probably easy enough to find and replace them all

Reply:
Though, I will say that lots of libraries use `LOG_<level>` as macros, so we may conflict with them... we may want a more unique prefix

Reply:
I made it `LOG_<LEVEL>` but to accommodate variadic arguments it's maybe a bit odd in that you need an internal set of parentheses... It's probably fine I guess, since it's internal logging.

Reply:
Hmm okay I see, probably not ideal indeed. This can be a solution for it:

```c++
#define LOG_CRITICAL(...) isobus::CANStackLogger::critical(__VA_ARGS__)
```

Only thing is that you can't explicitly state the needed arguments in the macro itself, we'd have to rely on doxygen for that. Maybe something like:
```c++
/// @param[in] logString A log statement
/// @param[in] args (optional) A list of printf style arguments to format the logString with
```

Reply:
Added your suggestion, though doxygen was mad about documenting `__VA_ARGS__`. I like having the `param` documentation so I surrounded that definition with `//! @cond Doxygen_Suppress`